data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+	on complete
+		"coalition jobs" ++
+		payment 470000
+		conversation
+			`Magister Lotuora meets you when you land, handing you your payment of <payment>, and congratulating you for managing to evade the pursuing ships. "Frustrated many of the Hunter pilots, I heard you have," she says with a laugh. "Good training for them, I hope this was. Finishing up the arrangements for one more drill, I am. When ready, you are, meet me in the spaceport, you must."`
+
+mission "Heliarch Drills 3"
+	name "Combat Drill"
+	description "Head to the <waypoints> system to fight the <npc>, reporting back to <planet> when you succeed. You must only disable the ship, destroying or capturing it would deem you an enemy of the Heliarchs."
+	source "Belug's Plunge"
+	waypoint "Torbab"
+	to offer
+		has "Heliarch Drills 2-C: done"
+		not "Resistance: Questions: done"
+	on offer
+		conversation
+			`Lotuora is speaking with a Kimek Heliarch captain at the entrance to the spaceport. "Ah, there the Captain is," she tells the Kimek. "Captain <last>, your sparring partner, this is," she says, gesturing to the Kimek, who salutes you with a short bow of her head. "Excited to test her ship against an outsider, she is."`
+			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, boarding a Punisher and flying away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall."`
+			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
+				accept
+	npc disable save
+		government "Heliarch Test Dummy"
+		personality disables staying nemesis heroic
+		system "Torbab"
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Punisher (Scrappy)"
+	on fail
+		"reputation: Heliarch" = -1000
+		"reputation: Coalition" = -1000
+	on complete
+		"coalition jobs" ++
+		"assisted heliarch" ++
+		payment 800000
+		conversation
+			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, waiting for a few seconds as she looks at you, as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`
+			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy, and wishes you safe travels, heading to a Heliarch building.`
```

Comment:
```suggestion
			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, heading to a Heliarch building.`
```